frontend/src/api/Firebase.ts

```diff
@@ -0,0 +1,13 @@
+import firebase from 'firebase/app';
+import 'firebase/auth';
+
+const app = firebase.initializeApp({
+  apiKey: process.env.REACT_APP_FIREBASE_API_KEY,
+  authDomain: process.env.REACT_APP_FIREBASE_DOMAIN,
+  databaseUrl: process.env.REACT_APP_FIREBASE_DATABASE,
+  projectId: process.env.REACT_APP_FIREBASE_PROJECT_ID,
+  storageBucket: process.env.REACT_APP_FIREBASE_STORAGE_BUCKET,
+  messagingSenderId: process.env.REACT_APP_FIREBASE_SENDER_ID,
```

Comment:
Sure I can do that - just wanted to bring up though if in the future we wanted to have different environments (similar to the Heroku Review Apps), this could be useful as we could [customize settings](https://create-react-app.dev/docs/deployment/#customizing-environment-variables-for-arbitrary-build-environments) based on the environment such as staging, production, etc. (in like `.env.staging` and `.env.production` files). Thoughts? That could be further in the future tho so I can revert it in the meantime if you prefer

Reply:
Got it - in that case I'm ok with keeping this file, if you think that's unlikely for the Firebase variables then move them directly into `Firebase.ts`, but if you think that's possible sure we can keep them separate.

Reply:
Sounds good, I'll keep them separate for now